.github/workflows/release_workflow.yml

```diff
@@ -0,0 +1,104 @@
+name: CAS Release Workflow
+
+# Controls when the action will run.
+# Triggers the workflow on push or pull request events
+on: [push, workflow_dispatch]
```

Comment:
This seems like it runs on any push or whenever another workflow runs?  But that doesn't match with the comment.